### PR TITLE
vendor: bump github.com/coreos/ignition/config

### PR DESCRIPTION
**Pull request by dghubble**
* Allow YAML unmarshaling configs

### CODE REVIEW

File: vendor.manifest [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# If you manipulate the contents of third_party/, amend this accordingly.
+# If you manipulate the contents of vendor/, amend this accordingly.
 # pkg							version
-github.com/alecthomas/units				6b4e7dc5e3143b85ea77909c72caf89416fc2915
```
Comment (dghubble) [inline vendor.manifest]: Is the intent to show just the dependencies vendored at this "level"?

Comment (crawford) [inline vendor.manifest]: Yes

**Comment by crawford** [comment]
The vendor path is wrong (which is why this patch is massive).

**Comment by dghubble** [comment]
Right, duplicate vendored ignition/config. Removed now

**Comment by crawford** [comment]
LGTM

**Pull request merged**